freqtrade/exchange/kraken.py

```diff
@@ -95,32 +108,91 @@ def stoploss(self, pair: str, amount: float, stop_price: float, order_types: Dic
 
         if self._config['dry_run']:
             dry_order = self.create_dry_run_order(
-                pair, ordertype, "sell", amount, stop_price)
+                pair, ordertype, side, amount, stop_price, leverage)
             return dry_order
 
         try:
             amount = self.amount_to_precision(pair, amount)
 
-            order = self._api.create_order(symbol=pair, type=ordertype, side='sell',
-                                           amount=amount, price=stop_price, params=params)
+            order = self._api.create_order(
+                symbol=pair,
+                type=ordertype,
+                side=side,
+                amount=amount,
+                price=stop_price,
+                leverage=leverage,
+                params=params
```

Comment:
I think leverage must be part of the `params` dict - not a direct parameter that's passed to ccxt.
while ccxt will (probably) not complain - it'll also not set leverage for this order correctly.